### PR TITLE
Shellcheck

### PR DESCRIPTION
**Pull request by wernerb**
These are minor changes to accommodate shellcheck and POSIX compatibility

### CODE REVIEW

**Comment by diogomonica** [comment]
This looks great. Let me just test this locally to make sure it doesn't break anything and I'll merge it.

Thank you for the contribution.

**Comment by wernerb** [comment]
Done! Thanks for your skill. This was a collaborative effort with @sgzijl.

**Comment by diogomonica** [comment]
@wernerb I still don't see the signed off.

Maybe you can squash all the commits into one and the use the Signed-off by https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work

The easiest way to do this is to amend the last commit:
```
$ git commit --amend -s --no-edit
$ git push -f
```

File: docker-bench-security.sh [hunk 1]
```diff
@@ -14,20 +14,18 @@
 . ./helper_lib.sh
 
 # Setup the paths
-this_path=$(abspath $0)       ## Path of this file including filenamel
-dir_name=`dirname ${this_path}`    ## Dir where this file is
-myname=`basename ${this_path}`     ## file name of this script.
-logger="${myname}.log"
-
+this_path=$(abspath "$0")       ## Path of this file including filenamel
+myname=$(basename "${this_path}")     ## file name of this script.
```
Comment (moviuro) [inline docker-bench-security.sh]: Missing quotes, see L17 too

**Comment by diogomonica** [comment]
@wernerb would love to merge it, if you can add Signed-off on your commit.

Thank you!

**Comment by wernerb** [comment]
Done! After this is merged, I will do a once-over again.

**Pull request merged**